### PR TITLE
Merge the launch_embedded into embedded

### PR DESCRIPTION
**Pull request by saebekassebil**
Hey guys,

This PR should include alle the stuff in the `launch_embedded` branch, and merge it into embedded in way that's more future-proof. (Settings instead of hardcoding etc).

@m-abs @dfg-nota Would you take a look at this?

### CODE REVIEW

**Comment by saebekassebil** [comment]
After merging we should put the `embedded` branch live, and delete the `launch_embedded` branch

File: Cakefile [hunk 1]
```diff
@@ -107,9 +107,9 @@ task "html", "Build HTML", (options) ->
   scripts = []
 
   if options.test
-    scripts.push "http://test.e17.dk/getnotaauthtoken"
+    scripts.push "http://nota.dk/getnotaauthtoken"
```
Comment (dfg-nota) [inline Cakefile]: There is also a [test.nota.dk](test.nota.dk) version, do we wanna use that?

File: src/config/config.coffee [hunk 2]
```diff
@@ -32,6 +32,11 @@ LYT.config =
       version:      "1"
       config:       null
 
+  navigation:
+    # Single quotes are important here - otherwise this gets interpreted
+    # as a template already be CoffeeScript
```
Comment (dfg-nota) [inline src/config/config.coffee]: Spelling: by CoffeeScript

**Comment by dfg-nota** [comment]
Looks good to me after comments

File: Cakefile (hunk 1 above)
Comment (saebekassebil) [inline Cakefile]: Maybe? I don't know. I just remember Michael saying that there was no test version at that point. More importantly: Do the test env have its own seperate session/login system or does it just reuse the live one?

**Comment by saebekassebil** [comment]
Spelling fixed.

File: Cakefile (hunk 1 above)
Comment (dfg-nota) [inline Cakefile]: Separate session and logins iirc. Server is also only available from the intranet.

Comment (saebekassebil) [inline Cakefile]: Not at a pc right now, but if this is all the test setting does, then we should use the test servers here as you propose.

Comment (dfg-nota) [inline Cakefile]: Should probably get this merged. Do you want to change the test setting before I do?

**Comment by saebekassebil** [comment]
@dfg-nota Fixed URLs. Ready for merge? (and push on live?)

**Comment by saebekassebil** [comment]
Cheers Daniel. Do you push this live or should I?

**Comment by dfg-nota** [comment]
I haven’t yet, feel free to do so.

Fra: Jakob Miland [mailto:notifications@github.com]
Sendt: 24. februar 2016 12:44
Til: Notalib/LYT <LYT@noreply.github.com>
Cc: Daniel Dam Freiling <dfg@Nota.dk>
Emne: Re: [LYT] Merge the launch_embedded into embedded (#692)


Cheers Daniel. Do you push this live or should I?

**Comment by saebekassebil** [comment]
Pushed live

**Pull request merged**